### PR TITLE
 Rename generateOffer to generateDataChannelOffer and remove use of legacy optionsRename generateOffer to generateDataChannelOffer and remove use of legacy options

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

**Review by alvestrand — APPROVED** [review]
Good catch that generateOffer() is only used to generate data channel offers!
Short helpers are good helpers.

File: webrtc/idlharness.https.window.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 'use strict';
 
 // The following helper functions are called from RTCPeerConnection-helper.js:
-//   generateAnswer()
+//   generateDataChannelAnswer()
```
Comment (soareschen) [inline webrtc/idlharness.https.window.js]: `generateAnswer()` is not renamed to `generateDataChannelAnswer()`.

**Review by soareschen — CHANGES_REQUESTED** [review]

File: webrtc/idlharness.https.window.js (hunk 1 above)
Comment (youennf) [reply webrtc/idlharness.https.window.js]: Fixed in above commit

**Review by soareschen — APPROVED** [review]

**Comment by youennf** [comment]
Firefox bot issue is probably unrelated, merging.

**Pull request merged**